src/frontend/src/handler/drop_schema.rs

```diff
@@ -62,7 +62,7 @@ pub async fn handle_drop_schema(
     };
     match mode {
         Some(DropMode::Restrict) | None => {
-            if let Some(table) = schema.iter_table().next() {
+            if let Some(table) = schema.iter_user_table().next() {
```

Comment:
This looks somehow wrong (but we prevented error somewhere else...)

```
dev=> create schema s;
dev=> create materialized view s.mv as select  1 as x;
dev=> drop schema s;
ERROR:  Failed to run the query

Caused by:
  Permission denied: PermissionDenied: The schema is not empty, try dropping them first


dev=> create table s.t (x int);
CREATE_TABLE
dev=> drop schema s;
ERROR:  Failed to run the query

Caused by these errors (recent errors listed first):
  1: Catalog error
  2: cannot drop schema s because table t depend on it
```

Reply:
Handled on meta here https://github.com/risingwavelabs/risingwave/blob/08fb7d4f847ebd2b870a9ae6ff082403c911de4e/src/meta/src/controller/utils.rs#L620-L634

Reply:
Can we simply remove the check on frontend? 🤔 cc @yezizp2012